scripts/codeCoverage.sh

```diff
@@ -1,69 +1,73 @@
 #!/bin/bash
 LAUNCH_DIR=`pwd`
+PROJECT="FileIO"
+OBJECT_DIR="${PROJECT}.dir"
 
-# If you need to add a header to the code coverage please put it here
-# example "myfile.h myotherfile.h"
+# --- WHITELIST ---
+# Add a header to the code coverage here
+#    Examples:
+#    HEADER_WHITELIST="myfile.h"
+#    HEADER_WHITELIST="myfile.h myotherfile.h"
 HEADER_WHITELIST=
 
-# black listing of files can also be done
-# please see commented out execution of gcovr at the bottom
-# example: 
-#  SOURCE_BLACKLIST=".*ProbeTransmogrifier.cpp.*"
-
+# --- BLACKLIST ---
+# Blacklisting of files can also be done
+#    Please see commented out execution of gcovr at the bottom.
+#    Example: 
+#       SOURCE_BLACKLIST=".*ProbeTransmogrifier.cpp.*"
+SOURCE_BLACKLIST="/usr/local/probe/*"
```

Comment:
Does this really work? 
1) does wildcard work?
2) does directory structure work? It's not what you wrote in the example, where the blacklist pointed to a concrete file
3) If directories and multiple files can be added with wildcard shouldn't still the path be
/usr/local/probe/include/*